lib/fastlane/helper/git_helper.rb

```diff
@@ -6,10 +6,11 @@ def self.git_log_between(pretty_format, from, to)
       nil
     end
 
-    def self.last_git_tag_name(match_lightweight = true)
+    def self.last_git_tag_name(match_lightweight = true, match_pattern = nil)
       command = ['git describe']
       command << '--tags' if match_lightweight
       command << '--abbrev=0'
+      command << "--match #{match_pattern}" if match_pattern
```

Comment:
I think you may need `${match_pattern.shellescape}` here, though it makes the value you need to test for in the `changelog_from_git_commits_spec` wind up double-escaped and fairly silly looking :smile: